test/Crowdsale.js

```diff
@@ -1,5 +1,8 @@
+import moment from 'moment'
```

Comment:
Given what we've discussed, we're probably going to remove moment.js entirely. Let's not add the dependency here.